main.go

```diff
@@ -126,6 +153,8 @@ func sendFunnyEmail(w http.ResponseWriter, r *http.Request) {
 	response, err := client.Send(message)
 	if err != nil {
 		log.Println(err)
+		w.WriteHeader(response.StatusCode)
+		json.NewEncoder(w).Encode(err.Error)
```

Comment:
When would you include `_ =` before `json.NewEncoder(w).Encode(err.Error)`? Only if you need to reference it again, right?

Reply:
_ is just ignoring the return the same as not putting it there it's just more explicitly saying we are potentially ignoring an error.